lib/logstash/outputs/elasticsearch.rb

```diff
@@ -178,6 +178,8 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   #
   # - index: indexes a document (an event from Logstash).
   # - delete: deletes a document by id
+  # - create_unless_exists: creates a document, raises exception if no id is provided
```

Comment:
@jordansissel No, that's why we changed it from `create` originally to `create_unless_exists` to drive home that it's different.

The functionality implied by `create_unless_exists` is still true in the HTTP client as long as you supply an ID with `create`. The only difference is that `create_unless_exists` requires you to specify the ID (as opposed to `create`, which will generate one if not supplied).